Schedule inflation of webhook-only mods once a day

HebaruSan opened the pull request:
## Motivation

In #40 we suppressed scheduling of modules that are adequately handled by webhook notifications.

In normal operation this helpfully reduces resource consumption, but sometimes we develop things like #70 that add functionality that depends on fresh inflations, which leaves the excluded modules behind. And there's always the possibility that we miss a webhook inflation due to some glitch on our side or SpaceDock's.

## Changes

Now modules that are excluded from normal passes will be scheduled for inflation once per day.

techman83 commented:
This would certainly run the scheduler every Monday at  2am. But there are a couple of things that I think we need to address.

- In hindsight there isn't a reason to schedule _all_ the netkans. I did have some code for inverting the logic based on a '--webhooks-only' flag. But it was messy and I haven't had a chance to get back and have a closer look.
- This may not fire at all as the status check is hardcoded to requiring 100 credits. This doesn't matter with the regular scheduling, but we're hoping we'll have spare credits when this schedule fires. A '--credits' flag was something I was also working with above. Default to 200, allow the webhooks to run with 100.
- I think we should run this daily, but that's a trivial implementation detail.

HebaruSan commented:
- Replaced the `--schedule-all` flag with a `--group` option that can be `webhooks`, `nonhooks`, or `all`
- Added a `--min-credits` option
- Changed to `rate(1 day)`

techman83 commented:
From my brief read over this looks good. I'll take a closer look after work and merge it in/update the stack.

techman83 commented on netkan/netkan/cli.py:
```diff
@@ -107,18 +107,27 @@ def indexer(queue, metadata, token, repo, user, key,
-    help='Schedule all modules even if we think they should be covered by webhooks'
+    '--group',
+    type=click.Choice(['all', 'webhooks', 'nonhooks']), default="nonhooks",
+    help='Which mods to schedule'
```
Click choices are really cool! :+1:

techman83 commented on netkan/netkan/scheduler.py:
```diff
@@ -25,29 +27,24 @@ def netkans(self):
+            return self.webhooks_group
+        else:
+            return self.nonhooks_group
+
```
I'm not overly sure about this logic, but we can add some tests for this and re-factor later if need be. Much better than what was springing to mind when I last had a look and this does the job. :+1:

techman83 commented on prod-stack.py:
```diff
@@ -330,7 +330,7 @@
 for task in [
-        'Scheduler', 'CertBot', 'StatusDumper', 'DownloadCounter']:
+        'Scheduler', 'SchedulerFullPass', 'CertBot', 'StatusDumper', 'DownloadCounter']:
     scheduler_resources.append(Sub(
```
I'll rename this before merging/deploying as it's not really a full pass any more.
